wasi-sdk.cmake

```diff
@@ -22,7 +22,6 @@ set(CMAKE_C_COMPILER_TARGET ${triple} CACHE STRING "wasi-sdk build")
 set(CMAKE_CXX_COMPILER_TARGET ${triple} CACHE STRING "wasi-sdk build")
 set(CMAKE_C_FLAGS "-v" CACHE STRING "wasi-sdk build")
 set(CMAKE_CXX_FLAGS "-v -std=c++11" CACHE STRING "wasi-sdk build")
```

Comment:
Should we remove these C flags as well?  Wouldn't it be best to leave it up individual projects to decide which C++ version they want.. .and doesn't `-v` generate a ton of verbose output?